omod/src/main/java/org/openmrs/module/patientdashboard/web/controller/OPDEntryController.java

```diff
@@ -253,7 +253,7 @@ public String firstView(
 			model.addAttribute("admittedStatus", "Admitted");
 		}
 		
-		IpdPatientAdmission ipdPatientAdmission=ipds.getIpdPatientAdmissionByPatient(patient);

```

Comment:
https://github.com/uonafya/hospitalcoreopenmrs/blob/master/api/src/main/java/org/openmrs/module/hospitalcore/IpdService.java#L128, ypou also have this in the hospital core